samza-core/src/main/java/org/apache/samza/runtime/LocalApplicationRunner.java

```diff
@@ -233,32 +214,41 @@ public void waitForFinish() {
    * @throws TimeoutException exception for latch timeout
    */
   /* package private */ void createStreams(String planId, List<StreamSpec> intStreams) throws TimeoutException {
-    if (!intStreams.isEmpty()) {
-      // Move the scope of coordination utils within stream creation to address long idle connection problem.
-      // Refer SAMZA-1385 for more details
-      CoordinationUtils coordinationUtils = createCoordinationUtils();
-      if (coordinationUtils != null) {
-        Latch initLatch = coordinationUtils.getLatch(1, planId);
-
-        try {
-          // check if the processor needs to go through leader election and stream creation
-          if (shouldContestInElectionForStreamCreation(initLatch)) {
-            LeaderElector leaderElector = coordinationUtils.getLeaderElector();
-            leaderElector.setLeaderElectorListener(() -> {
-                getStreamManager().createStreams(intStreams);
-                initLatch.countDown();
-              });
-            leaderElector.tryBecomeLeader();
-            initLatch.await(LATCH_TIMEOUT_MINUTES, TimeUnit.MINUTES);
-          }
-        } finally {
-          coordinationUtils.reset();
-        }
-      } else {
-        // each application process will try creating the streams, which
-        // requires stream creation to be idempotent
+    if (intStreams.isEmpty()) {
+      LOG.info("Set of intermediate streams is empty. Nothing to create.");
+      System.out.println("1");
```

Comment:
interesting output :). There are a bunch of System.out below which need to be removed.

Reply:
system.outs will be removed.